bin/find-graphql-files.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+GRAPHQL_FILE_LIST="$DERIVED_FILE_DIR/GraphQLFiles.xcfilelist"
+
+#Find all GraphQL files in KsApi.
+#Replace the literal $SRCROOT with the token $(SRCROOT)
+find $SRCROOT -name "*.graphql" | sed "s#^$SRCROOT#\$(SRCROOT)#" > "$GRAPHQL_FILE_LIST.tmp"
```

Comment:
This replace step isn't strictly necessary, but when I was debugging, I found it odd to have a list of fully qualified files in it. I like this slightly better. 🤷‍♀️